src/EventService.Data.Provider.MsSql.Ef/Migrations/20230126193800_InitialTables.cs

```diff
@@ -151,6 +151,24 @@ private void CreateEventFilesTable(MigrationBuilder migrationBuilder)
       {
         table.PrimaryKey($"PK_{DbEventFile.TableName}", ef => ef.Id);
       });
+  }
+
+  private void CreateUsersBitrthdaysTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbUserBirthday.TableName,
+      columns: table => new
+      {
+        UserId = table.Column<Guid>(nullable: false),
+        DateOfBirthday = table.Column<DateTime>(nullable: false),
+        IsActive = table.Column<bool>(nullable: false),
+        CreatedAtUtc = table.Column<DateTime>(nullable: false),
+        ModifiedAtUtc = table.Column<DateTime>(nullable: true)
+      },
```

Comment:
so, is it the separate event, that is different from dbEvent and has no relations with it?
or it will be connected with dbEvents in the future?

Reply:
Yes? it is separate